environments/Test-Laptop.json

```diff
@@ -41,6 +41,9 @@
         "hdp_utils": "http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.20/repos/ubuntu12"
       },
       "hadoop" : {
+        "environment" : {
+          "profile" : ""
```

Comment:
Either here or probably better on the source issue #484, please expand on how profile will be used.  I am starting to think, that perhaps more then a single profile may be a better approach such that we can have 
`profiles = [ "HBase_over_transatlantic_links",  "vm_install", "etc" ]`